test/framework/cloudstack.go

```diff
@@ -298,6 +343,21 @@ func (c *CloudStack) Redhat124Template() api.CloudStackFiller {
 	return api.WithCloudStackStringFromEnvVar(cloudstackTemplateRedhat124Var, api.WithCloudStackTemplateForAllMachines)
 }
 
+// Redhat125Template returns cloudstack filler for 1.24 Ubuntu.
+func (c *CloudStack) Redhat125Template() api.CloudStackFiller {
+	return api.WithCloudStackStringFromEnvVar(cloudstackTemplateRedhat125Var, api.WithCloudStackTemplateForAllMachines)
+}
+
+// Redhat126Template returns cloudstack filler for 1.24 Ubuntu.
+func (c *CloudStack) Redhat126Template() api.CloudStackFiller {
+	return api.WithCloudStackStringFromEnvVar(cloudstackTemplateRedhat126Var, api.WithCloudStackTemplateForAllMachines)
+}
+
+// Redhat127Template returns cloudstack filler for 1.24 Ubuntu.
+func (c *CloudStack) Redhat127Template() api.CloudStackFiller {
+	return api.WithCloudStackStringFromEnvVar(cloudstackTemplateRedhat127Var, api.WithCloudStackTemplateForAllMachines)
+}
```

Comment:
I would remove these and just use the `UpdateRedhatTemplate12XVar` function in e2e